client/state/posts/test/utils.js

```diff
@@ -141,4 +143,82 @@ describe( 'utils', () => {
 			} );
 		} );
 	} );
+
+	describe( '#getTermIdsFromEdits()', () => {
+		it( 'should return the same post edit object if no term edits have been made', () => {
+			const normalizedPostEdits = getTermIdsFromEdits( {
```

Comment:
Minor: Could improve "returns same" test to compare the references (`equal` vs. `eql` / `deep.equal`):

```
const original = { title: 'Chewbacca Saves' };
const normalizedPostEdits = getTermIdsFromEdits( original );

expect( normalizedPostEdits ).to.equal( original );
```